tests/tasks/setup/eks/awscli-cp-with-vpc.yaml

```diff
@@ -78,4 +78,12 @@ spec:
       # install csi drivers.
       kubectl apply -k "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=$(params.aws-ebs-csi-driver-version)"
       # TODO: Calculate replicas based on the cluster size going forward.
-      kubectl scale --replicas 20 deploy coredns -n kube-system
\ No newline at end of file
+      kubectl scale --replicas 1000 deploy coredns -n kube-system
```

Comment:
20 -> 1000 ? hoping its not over provisioning.  I think we had a backlog item to calculate it dynamically based on the node size, shall we implement it

Reply:
That's why we still have a To-do, this requires a discussion w N/w team too on what's the right number, I'm increasing now because we are like 10X worse w these numbers.